cluster/saltbase/salt/kube-master-addons/kube-master-addons.sh

```diff
@@ -45,15 +45,24 @@ while true; do
     elif [[ $rc == 124 ]]; then
       restart_docker=true
     fi
+
+    timeout 30 docker load -i /srv/salt/kube-bins/federated-apiserver.tar 1>/dev/null 2>&1
```

Comment:
I am not sure this is an acceptable way to get the container onto nodes. Am thinking about it.

There's two problems with this:
1. it distributes this binary to every master & node, even for users that don't want to run it
2. you can't ship an updated version of the container without tearing the cluster down?